### PR TITLE
[NIFI-13142] configure typography

### PR DESCRIPTION
**Pull request by scottyaslan**
Theme files now configure their font-family and can customize their typography. To demonstrate the art of the possible the default material theme uses the 'Roboto' font while the purple theme has been updated to use the standard 'Courier New' font.

Default material theme with 'Roboto' font:

Screenshot 2024-05-04 at 11 55 11 AM
Screenshot 2024-05-04 at 11 55 24 AM

Purple theme with 'Comic sans' font:

Screenshot 2024-05-06 at 2 40 19 PM
Screenshot 2024-05-06 at 2 40 27 PM

The nifi theme file is configured with 'Roboto Slab'. This can also be changed in the nifi.scss file to make use of different fonts.

### CODE REVIEW

**Comment by rfellows** [comment]
will review...

File: nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-frontend/src/main/nifi/src/styles.scss [hunk 1]
```diff
@@ -54,7 +50,7 @@
 // To override the canvas theme, you need to set the variables $nifi-theme-light and $nifi-theme-dark
 @import 'assets/themes/nifi';
 // To override the NiFi theme, you need to set the variables $material-theme-light and $material-theme-dark
-@import 'assets/themes/material';
+@import 'assets/themes/purple';
```
Comment (james-elliott) [inline nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-frontend/src/main/nifi/src/styles.scss]: I'm sure you meant to leave this as the Material theme by default

**Review by james-elliott — CHANGES_REQUESTED** [review]

File: nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-frontend/src/main/nifi/src/styles.scss (hunk 1 above)
Comment (scottyaslan) [reply nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-frontend/src/main/nifi/src/styles.scss]: Already fixed my dude!

**Review by rfellows — APPROVED** [review]
These changes really make the UI super clean IMO. I love how all of the various fonts used have come together into 1. The changes to the "purple" theme with the Comic Sans font are great. It really shows off the power of the theming and typography.

Well done @scottyaslan. I'll merge this once CI completes.

**Pull request merged**